make-yamls.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#Find all template files
+#substitute the volume directory and then emit a yaml file
+for tmpl in *.tmpl; do
```

Comment:
I had to use this because the copy + template rewrite stuff in gradle doesn't work with YAMLs. It's looking for java properties files, unfortunately.

Reply:
That's fine. The gradle copy/replace task works on all sorts of files and ways to "express" template values. Not just "groovy properties". I can show you or if it is necessary to have everything grooving in gradle. 

I only did this sort of thing in gradle for the fair-testing because there is already lots of gradle. So, I did the substitution that way, instead of a bash script.

BTW, what I did in that project the gradle task could be used here in place of this script. Again, not a biggie.

Reply:
I copied over that exact task and it didn't work. It kept throwing out parsing errors on the first line of the yaml file. It was choking on the `:`, since I guess it uses that as part of the GString expansion. Anyways, I was able to get this up and working in 5 minutes.